float32 field1
float32 field2
float32 field3
float32 field4
float32 field5
bool field6

